roles/addons/diskless/files/disklessset.py

```diff
@@ -20,6 +20,7 @@
 
 import yaml
 from ClusterShell.NodeSet import NodeSet
+from subprocess import *
```

Comment:
You only need check_call from subprocess. Maybe import this one only ?

Reply:
Done.